packages/playground/src/components/DemoFrame.tsx

```diff
@@ -53,16 +53,6 @@ export default function DemoFrame(props: DemoFrameProps) {
 
   const instanceRef = useRef<any>();
 
-  const handleRef = useCallback(
-    (ref: any) => {
-      instanceRef.current = {
-        contentDocument: ref ? ref.node.contentDocument : null,
-        contentWindow: ref ? ref.node.contentWindow : null,
-      };
-    },
-    [instanceRef],
-  );
```

Comment:
not needed anymore?

Reply:
No, it the `instanceRef` is now directly set and does the same thing